common/primitives/src/msa.rs

```diff
@@ -8,6 +8,9 @@ use sp_std::prelude::Vec;
 
 pub use crate::{ds::OrderedSetExt, schema::SchemaId};
 
+/// The gap between the current block and a future expiring block allowed when validating signature proofs.
+pub const EXPIRATION_BLOCK_VALIDITY_GAP: u32 = 128;
```

Comment:
nit: should this number be selected in a way that we get some round number of minutes considering that each block might be 6 or 12 seconds. it looks like right now with 6 sec per block we get 12.8 minutes which is a bit non round.

Reply:
@johnmela What is the default mortality on transactions?